pkg/util/iptables/iptables.go

```diff
@@ -783,3 +802,75 @@ func isResourceError(err error) bool {
 	}
 	return false
 }
+
+type ParseError interface {
+	Line() int
+	Error() error
+}
+
+type parseError struct {
+	cmd  string
+	line int
+}
+
+func (e parseError) Line() int {
+	return e.line
+}
+
+const (
+	colon = ":"
```

Comment:
This adds negative value, I think.  Just say `":"` at the call site :)

Reply:
yeah :)